locksmith/app.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express')
+var cors = require('cors');
+var bodyParser = require('body-parser');
+const models = require('./models')
+const app = express()
+const Lock = require('./sequelize')
+
+var router = express.Router();
+
+router.route('/lock').put(function (req, res, next) {
```

Comment:
I wonder if it is actually even worth "replacing" the name for a lock? Couldn't we just add a new record with the deployed address (and keep the old one even if never used ever again).

Reply:
This is a really good point!

Reply:
Careful about primary key here though - I agree with using `address`, but we'd want to make sure the new record with the stored name was the one that was always retrieved.